nengo/builder/connection.py

```diff
@@ -80,7 +80,11 @@ def build_decoders(model, conn, rng, transform):
         targets = multiply(targets, transform.T)
 
     try:
-        wrapped_solver = model.decoder_cache.wrap_solver(solve_for_decoders)
+        if model.seeded[conn]:
```

Comment:
I'm trying to figure out how `model.seeded` works. Doesn't it mean that if only the top level network is seeded, connections in sub-networks won't be cached?